package.json

```diff
@@ -33,25 +34,24 @@
   "author": "brian@hovercraftstudios.com",
   "license": "MIT",
   "devDependencies": {
-    "@most/eslint-config-most": "^1.0.3",
-    "assert": "^1.4.1",
+    "@briancavalier/assert": "^3.4.0",
+    "@most/scheduler": "^0.13.0",
     "buba": "^4.0.1",
-    "eslint": "^3.11.1",
-    "eslint-config-standard": "^5.3.5",
-    "eslint-plugin-promise": "^2.0.1",
-    "eslint-plugin-standard": "^2.0.1",
-    "istanbul": "^1.1.0-alpha.1",
+    "cpy-cli": "^1.0.1",
+    "flow-bin": "^0.54.1",
     "jsinspect": "^0.8.0",
     "mocha": "^2.5.3",
     "most": "^1.1.0",
```

Comment:
Do we need this old `most`?

Reply:
Weird, I thought I removed that :/. Thanks!